internal/secrets/import.go

```diff
@@ -55,10 +55,13 @@ func ImportClusterIfEmpty(silent bool) (pkgtypes.Cluster, error) {
 	log.Info().Msgf("import cluster secret discovered for cluster %s", cluster.ClusterName)
 
 	// if you find a record bail
-	_, err = GetCluster(kcfg.Clientset, cluster.ClusterName)
+	existingCluster, err := GetCluster(kcfg.Clientset, cluster.ClusterName)
 	if err != nil {
-		log.Info().Stack().Msgf("did not find preexisting record for cluster %s. importing record.", cluster.ClusterName)
+		return existingCluster, fmt.Errorf("unable to find cluster: %s", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("unable to find cluster: %w", err)
```

Make sure to wrap errors, not just `%s` them, otherwise they become uncomparable.

Reply:
I'm validating against that object return https://github.com/konstructio/kubefirst-api/blob/fix-import/internal/secrets/import.go#L63, is that going to turn that nil object o an empty object?

Reply:
nil ig